components/mixmax/package.json

```
@@ -12,5 +12,8 @@
   "license": "MIT",
   "publishConfig": {
     "access": "public"
+  },
+  "dependencies": {
+    "@pipedream/platform": "^1.1.1"
```

I guess pnpm will still use `1.2.1` but can we change it `1.2.1` as it is the latest version?
```suggestion
    "@pipedream/platform": "^1.2.1"
```